backend/Application/DTOs/Admin/UpdateAdminDTos.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.DTOs.Admin
+{
+    public class UpdateAdminDTos
+    {
```

Comment:
For now let's not create Update methods, we may have different patch methods that updates different thinks, like one that updates the email, another will updates the UserName, etc... but we may add that in the future once it's needed by the frontend, for now we can just have Register, Remove, GetAll and GetById endpoints.